perllib/FixMyStreet/Cobrand/EastSussex.pm

```diff
@@ -6,4 +6,37 @@ use warnings;
 
 sub council_area_id { return 2224; }
 
+sub open311_pre_send {
+    my ($self, $row, $open311) = @_;
+
+    my $contact = $row->category_row;
+    my $fields = $contact->get_extra_fields;
+    for my $field ( @$fields ) {
+        if ($field->{variable} && !$field->{automated}) {
+            my $text = $row->detail;
+            my $q = $row->get_extra_field_value( $field->{code} );
+            $text .= "\n\n" . $field->{description} . "\n" . $q;
+            $row->detail($text);
+        }
+    }
+}
+
+sub open311_post_send {
+    my ($self, $row, $h, $contact) = @_;
+
+    $contact = $row->category_row;
```

Comment:
It's slightly confusing having `$contact` as an argument and then ignoring it but reusing the variable. Is there a reason for doing it like that?